clang/include/clang/AST/ASTNodeTraverser.h

```diff
@@ -11,6 +11,22 @@
 // similar to RecursiveASTVisitor.
 //
 //===----------------------------------------------------------------------===//
+//
+// Modifications to this file by SEI staff are copyright Carnegie Mellon
+// University and contributed under the Apache License v2.0 with LLVM
+// Exceptions.
+//
+// SEI Contributions are made with funding sand support from the Department of
```

Comment:
```suggestion
// SEI Contributions are made with funding and support from the Department of
```

Reply:
Typo, but we probably can't allow this copyright notice anyway, per project policy.